modules/virt-example-vm-node-placement-node-affinity.adoc

```diff
@@ -6,11 +6,13 @@
 [id="virt-example-vm-node-placement-node-affinity_{context}"]
 = Example: VM node placement with node affinity
 
+[role="_abstract"]
 In this example, the VM must be scheduled on a node that has the label `example.io/example-key = example-value-1` or the label `example.io/example-key = example-value-2`. The constraint is met if only one of the labels is present on the node. If neither label is present, the VM is not scheduled.
 
 If possible, the scheduler avoids nodes that have the label `example-node-label-key = example-node-label-value`. However, if all candidate nodes have this label, the scheduler ignores this constraint.
 
 .Example VM manifest
```

Comment:
I'm guessing this is an example of `In DITA 1.3, only the <example>, <fig>, and <table> elements can have a title.` 

Still getting used to the block title exceptions. :)